src/pages/category/Category.js

```diff
@@ -1,9 +1,56 @@
-import React from 'react'
+import React from 'react';
+import './Category.scss';
 
 const Category = () => {
   return (
-    <div>Category</div>
-  )
-}
+    <div className="category">
+      <div className="categoryImg">
+        <img src="/images/meat.jpg"></img>
+      </div>
+      <ul className="categoryBtn">
+        <li>돼지</li>
+        <li>소</li>
+        <li>닭</li>
+        <li>수산</li>
+        <li>밀키트</li>
+        <li>우유</li>
+        <li>달걀</li>
+        <li>이유식</li>
+      </ul>
+      <div className="categoryItemBox">
+        <ul className="categoryItem">
+          <li>
+            <img src="/images/meat.jpg"></img>
```

Comment:
```suggestion
            <img src="/images/meat.jpg" />
```

안에 내용이 없는 태그들은 바로 닫아야합니다.
다른 모든 img 태그에도 적용해주세요.

Reply:
반영했습니다.